checklist.md

```diff
@@ -0,0 +1,20 @@
+# Checklists
+
+## Suspending Someone
+
+1. Suspend on GitHub
+2. Suspend on Matrix with mjolnir
+3. Suspend on Discourse
+4. Remove inappropriate content
+5. Add entry to moderation-log.md
+  - Include the length of the suspension, if not indefinite
+  - Do __not__ include why the suspension was issued
```

Comment:
We could include the broader category as a reason. Everybody can have a bad day or can fucks up from time to time, it doesn't have to be a big deal to get suspended.